Documents/Documents/Code/Controllers/Filters/ASCFiltersViewController.swift

```diff
@@ -9,72 +9,92 @@
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Documents", comment: ""), isSelected: false),
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Presentations", comment: ""), isSelected: false),
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Spreadsheets", comment: ""), isSelected: false),
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Images", comment: ""), isSelected: false),
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Media", comment: ""), isSelected: false),
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Archives", comment: ""), isSelected: false),
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("All files", comment: ""), isSelected: false),
-                                 ]),
-        ASCDocumentsSectionModel(sectionName: NSLocalizedString("Author", comment: ""),
-                                 filters: [
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Users", comment: ""), isSelected: false),
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Groups", comment: ""), isSelected: false),
-                                 ]),
-        ASCDocumentsSectionModel(sectionName: NSLocalizedString("Search", comment: ""),
-                                 filters: [
-                                     ASCDocumentsFilterModel(filterName: NSLocalizedString("Exclude subfolders", comment: ""), isSelected: false),
-                                 ]),
+        ASCDocumentsSectionModel(
+            sectionName: NSLocalizedString("Type", comment: ""),
+            filters: [
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("Folders", comment: ""), isSelected: false),
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("Documents", comment: ""), isSelected: false),
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("Presentations", comment: ""), isSelected: false),
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("Spreadsheets", comment: ""), isSelected: false),
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("Images", comment: ""), isSelected: false),
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("Media", comment: ""), isSelected: false),
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("Archives", comment: ""), isSelected: false),
+                ASCDocumentsFilterModel(filterName: NSLocalizedString("All files", comment: ""), isSelected: false),
+            ]
+        ),
+        ASCDocumentsSectionModel(
```

Comment:
Make a protocol with one method - return ASCDocumentsFilterModel list
Then the class that will implement it. For example ASCDocumentsFixedFilterModelsProvider.
The current class should receive this provider as a service in the constructor or in a property.
They can be divided into sections by an additional attribute distinguishing them by groups